src/libs/routing/routes/strategyCreateEdit.ts

```diff
@@ -15,22 +15,21 @@ export interface StrategyCreateSearch {
   strategyType?: StrategyType;
   strategyDirection?: StrategyDirection;
   strategySettings?: StrategySettings;
-  strategy?: string;
+  buyMin?: string;
+  buyMax?: string;
+  buyBudget?: string;
+  sellMin?: string;
+  sellMax?: string;
+  sellBudget?: string;
 }
 
 export const createStrategyPage = new Route({
   getParentRoute: () => rootRoute,
   path: '/strategies/create',
   component: CreateStrategyPage,
-  validateSearch: (search: Record<string, unknown>): StrategyCreateSearch => {
-    if (
-      search.strategyType === 'recurring' ||
-      search.strategyType === 'disposable' ||
-      search.strategy
-    ) {
-      return search;
-    }
-    return { ...search, strategyType: 'recurring' };
+  validateSearch: (search: StrategyCreateSearch = {}): StrategyCreateSearch => {
+    search.strategyType ||= 'recurring';
+    return search;
```

Comment:
passing the search like this will not filter out any additional search params.

https://issue--1015.carbon-app-csq.pages.dev/strategies/edit/1020847100762815390390123822295304634371?type=editPrices&test=123

```ts
return { 
    strategyType: search.strategyType || 'recurring',
    buyMin: roundNumber(formatNumber(sanitizeNumber(search.buyMin))) || undefined
    // ... and same for all other keys
}
```